server/models/sync-events.js

```diff
@@ -0,0 +1,122 @@
+import { resources } from "../../scripts/neuron-pipe/consts";
+import Listen from "../libs/listen";
+import { to, waitForEach } from "@libs/promise";
+import * as resourceProcessors from "../../scripts/neuron-pipe/resources";
+import config from "../libs/config";
+
+const token = config.SEARCH_APP_SERVICE_TOKEN;
+
+const fakeEvent = {
+  "_createdAt": Date.now(),
+  "status": "pending",
+  "resourceTypesToSync": [
+    "mr"
+  ],
+  "resources": [
+    {
+      "resourceType": resources["mr"],
+      "@id": "https://bbp-nexus.epfl.ch/staging/v0/data/thalamusproject/simulation/morphologyrelease/v0.1.2/1cb0b760-2935-4aa8-971a-cdac17d9c4a5",
```

Comment:
Shouldn't we test against production resources?

Reply:
Unfortunately some of the resource types only live in staging for the moment